strabo/templates/private/upload_images.html

```diff
@@ -24,19 +24,15 @@
             </div>
 
             <div class="form-group">
-            <label for="year">Date Taken:</label><br>
+            <label for="year">Date Taken [mm]-[yyyy]:</label><br>
             <i>Images are ordered by date when displayed.</i>
```

Comment:
Yeah, that is programming language creeping into the UI there. Whoops.

My goal here is to tell the administrator that images are displayed in a narrative fashion, where images with earlier dates are displayed before those with later dates. This is so that they know that dates do matter, even if they don't directly display anywhere.

How about just that: "Images are displayed in a narrative fashion; those with earlier dates are displayed before those with later dates."